src/transformers/models/blip_2/modeling_blip_2.py

```diff
@@ -84,6 +84,104 @@ def to_tuple(self) -> Tuple[Any]:
         )
 
 
+@dataclass
+# Copied from transformers.models.clip.modeling_clip.CLIPOutput with CLIP->Blip2ImageTextMatchingModel
+class Blip2ImageTextMatchingModelOutput(ModelOutput):
+    """
+    Args:
+        loss (`torch.FloatTensor` of shape `(1,)`, *optional*, returned when `return_loss` is `True`):
+            Contrastive loss for image-text similarity.
+        logits_per_image:(`torch.FloatTensor` of shape `(image_batch_size, text_batch_size)`):
+            The scaled dot product scores between `image_embeds` and `text_embeds`. This represents the image-text
+            similarity scores.
+        logits_per_text:(`torch.FloatTensor` of shape `(text_batch_size, image_batch_size)`):
+            The scaled dot product scores between `text_embeds` and `image_embeds`. This represents the text-image
+            similarity scores.
```

Comment:
```suggestion
        logits_per_image (`torch.FloatTensor` of shape `(image_batch_size, text_batch_size)`):
            The scaled dot product scores between `image_embeds` and `text_embeds`. This represents the image-text
            similarity scores.
        logits_per_text (`torch.FloatTensor` of shape `(text_batch_size, image_batch_size)`):
            The scaled dot product scores between `text_embeds` and `image_embeds`. This represents the text-image
            similarity scores.
```
This is a small typo in the CLIP docstring, feel free to run `make fix-copies` to fix it there as well

Reply:
CLIP docstring type was fixed, and the copies updated